common/app/model/content/Atoms.scala

```diff
@@ -374,6 +414,39 @@ object Quiz extends common.Logging {
   }
 }
 
+object StoryQuiz {
+  def extractImage(quiz: atomapi.quiz.QuizAtom): Option[String] = for {
+    firstQuestion <- quiz.content.questions.headOption
+    firstAsset <- firstQuestion.assets.headOption
+    json = Json.parse(firstAsset.data)
+    assets = json \ "assets"
+    widths = (assets \\ "width").map(_.as[Int])
+    width140 = widths.indexWhere(_ == 140)
+    url <- (assets(width140) \ "secureUrl").asOpt[String]
```

Comment:
"This is temporary" are some famous last words. 😋 
It's your feature, you make the call but I agree with you this should not stay permanently in the codebase in this state. 
Scala looks fine otherwise, feel free to merge.